server_py/flatgov/templates/bills/detail.html

```diff
@@ -286,141 +301,184 @@ <h4>Understand the Context</h4>
+                View all CRS Reports for {{ bill.type_abbrev }} {{ bill.meta.number }} in the {{ bill.meta.congress }}th Congress
             </h6>
-
-            <table class="table mt-3" id='statements-tables'>
+            <table class="table mt-3">
                 <thead>
                     <tr>
-                        <th scope="col">Title</th>
-                        <th scope="col">Date</th>
-                        <th scope="col">Link</th>
+                        <th scope="col">Co-Sponsor</th>
+                        <th scope="col">Committee Jurisdiction</th>
+                        <th scope="col">Party</th>
+                        <th scope="col">Rank</th>
                     </tr>
                 </thead>
                 <tbody>
-                    {% for crs_report in crs_reports %}
-
                     <tr>
-                        <td style="width: 300px;">{{ crs_report.title|truncatechars:120 }}</td>
-                        <td>{{ crs_report.date }}</td>
-                        <td>
-                        {% if crs_report.link %}
-                        <a href='{{ crs_report.link }}' target="_blank" class='btn btn-primary'>
-                            Full Report
-                        </a>
-                        {% else %}
-                          [No link to report]
-                        {% endif %}
-                        </td>
+                        <td>Raúl Grijalva</td>
+                        <td>No</td>
+                        <td>D</td>
+                        <td>No</td>
                     </tr>
-
-                    {% endfor %}
                     <tr>
-                        <td>
-
-                            {% if not crs_reports %}
-                                No data available in table.
-                            {% endif %}
-                        </td>
+                        <td>Rodney Davis</td>
+                        <td>House Administration</td>
+                        <td>R</td>
+                        <td>Ranking Member</td>
+                    </tr>
+                    <tr>
+                        <td>Jim Cooper</td>
+                        <td>House Oversight</td>
+                        <td>D</td>
+                        <td>No</td>
+                    </tr>
+                    <tr>
+                        <td>Virginia Foxx</td>
+                        <td>House Oversight</td>
+                        <td>R</td>
+                        <td>No</td>
                     </tr>
                 </tbody>
             </table>
-        </div>
+        </div>    
+
         <div class="tab-pane fade" id="statement-admin" role="tabpanel" aria-labelledby="statement-admin-tab" style="min-height:20vh; max-height: 50vh; overflow: scroll;">
+                <h6 class="pt-3">
+                    View all the Statements of Administration Policy of {{ bill.type_abbrev }} {{ bill.meta.number }} in the {{ bill.meta.congres }}th Congress
```

Comment:
is this a typo? congres

one reason to avoid shortening words is to make it easy to read code

Reply:
This line looks like the error referenced in 
#196

Is it because we're using meta.? I see it in other places too, let's make sure we are correctly referencing in all instances